apm-agent-plugins/apm-opentelemetry/apm-opentelemetry-plugin/src/test/java/co/elastic/apm/agent/opentelemetry/tracing/ElasticOpenTelemetryTest.java

```diff
@@ -706,4 +782,10 @@ public String get(@Nullable Map<String, String> carrier, String key) {
             return carrier.get(key);
         }
     }
+
+    @Target(ElementType.PARAMETER)
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface TestAnnotation {
```

Comment:
[question] Maybe I'm missing something, but I don't see what this custom annotation helps testing.

Reply:
@Hi @SylvainJuge, first I tested only with single `SpanAttribute` annotation, then I added another annotation type and my test failed. So, I added this annotation to check that instrumentation is working with different annotations order.